### ISSUE TITLE
Rounded start menu makes the start menu become fully transparent sometimes

### ISSUE THREAD

**Issue by happymimimix** [valinet/ExplorerPatcher#4065]
### Before reporting your issue

- [X] I have confirmed that this issue does not happen when ExplorerPatcher is not installed
- [X] I do not have "register as shell extension" enabled
- [X] I have tried my best to check existing issues

### Repro ExplorerPatcher versions

The current latest

### Repro Windows Versions

Windows Server 2022 build 20348.587
fe_release.210507-1500

### 3rd party tweak software installed

The current latest version of rectify 11

### Describe the bug

Enable rounded, floating start menu in explorer patcher. 
Do some other things, opening wordpad, mspaint, etc. 
Then open the start menu. 
It becomes fully transparent with no background. 

### Expected outcome

Always correctly shows the start menu background with rounded corners no matter what tasks I do. 

### Actual outcome

When running mspaint and wordpad or some other programs, there's about 70% chance the start menu will become fully transparent with no background. 
![image](https://github.com/user-attachments/assets/84492670-73b7-4413-94c0-76823acb8937)

**Comment by pyrates999** [comment]
Does the transparent start menu still happen with rectify 11 uninstalled?

**Comment by Amrsatrio** [comment]
20348 is not one of our fully supported builds but we'll keep this as a reference.

**Comment by happymimimix** [comment]
> 20348 is not one of our fully supported builds but we'll keep this as a reference.

This is windows server 2022.

**Comment by happymimimix** [comment]
> Does the transparent start menu still happen with rectify 11 uninstalled?

Yes, it does. I've already tried uninstalling rectify 11 and it still happens.

**Comment by pyrates999** [comment]
Thank you for confirming rectify 11 is not the cause of this.
